frontend/public/components/_dropdown.scss

```diff
@@ -180,7 +181,7 @@ $dropdown-link-hover-border: 1px solid #bee1f4;
 }
 
 .co-namespace-selector {
-  max-width: 60%;
+  max-width: 100%;
```

Comment:
This was 60% before because when on mobile it would shrink the project dropdown and use ellipsis to keep it and the import yaml button on a single line.
With the addition of the app selector trying to fit all on one line is not feasible.
What I could do actually is add a class to change the behavior only for when the app selector is included in this bar. I would pass in an additional className to apply for our use case.

That way we maintain the original behavior:
![image](https://user-images.githubusercontent.com/14068621/59518196-2c88a480-8e93-11e9-8eb6-2435e42c4fa8.png)

While with the app selector we get this:
![image](https://user-images.githubusercontent.com/14068621/59518222-3ca08400-8e93-11e9-9ce2-79633b8f3f0a.png)